src/service/api/articles.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const {Router} = require(`express`);
+const {HttpCode} = require(`../../constants`);
+const articleExists = require(`../middlewares/article-exists`);
+const articleValidator = require(`../middlewares/article-validator`);
+const commentExists = require(`../middlewares/comment-exists`);
+const commentValidator = require(`../middlewares/comment-validator`);
+const routeParamsValidator = require(`../middlewares/route-params-validator`);
+
+const router = new Router();
+
+module.exports = (app, articleService, commentService) => {
+  app.use(`/articles`, router);
```

Comment:
```suggestion

module.exports = (app, articleService, commentService) => {
const router = new Router();

  app.use(`/articles`, router);
```

Сделай такое на всех роутах. Нужно чтобы все роуты генерировали новые пути через функцию. Дальше без этого будет сложно, и будет лишние непонятки.

Reply:
+